### PR TITLE
Add real session data

### PR DESCRIPTION
**Pull request by andresag4**
## Description

Load production seed file:
```
 rake db:prod_seed
 ```

To specify a defined starting date, send number of month and day as following
```
 rake "db:prod_seed[9,26]"
```

## How has this been tested?

Please mark the tests that you ran to verify your changes. If difficult to test, consider providing instructions so reviewers can test.

- [x] Manual testing
- [ ] System tests
- [ ] Unit tests
- [ ] None

## Checklist

- [x] CI pipeline is passing
- [x] My code follows the conventions of this project
- [x] I have performed a self-review of my code
- [ ] I have commented on my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation (if applicable)
- [ ] I have added seed data to the database (if applicable)

## Release tasks

Add any tasks that need to be done before/after the release of this feature.

## Screenshots/Loom

This section is relevant in case we want to share progress with the team, otherwise, it can be omitted.

### CODE REVIEW

**Comment by Sergio-e** [comment]
I suggest moving the images to the assets folder

**Comment by Sergio-e** [comment]
I'm finding some gaps on the sessions#show and the speaker#show pages. I suggest fixing them in a separate PR (@LuigiR0jas, @EdgarB).

#### Sessions#show
- let's remove the extra gap if the session doesn't have any tags or a description
- clicking on the speaker isn't redirecting to the speakers#show page
- the bookmark icon isn't working
- the "About this speaker" section doesn't seem to have the right width
- clicking on the speaker isn't redirecting to the speakers#show page
- some images aren't being correctly displayed
- shouldn't the session title be in red as in the sessions#index? I feel it should be the same on both places
image

image

image

**Comment by Sergio-e** [comment]
cc: @EdgarB @LuigiR0jas 
#### Speakers#show
- In some cases, there's a strange gap between the social media buttons and the upper/lower elements. In this case, the buttons are closer to the "Sessions by this speaker" element than to the speakers description, which looks weird
image
- In some cases, the "Sessions by this speaker" isn't taking the whole width
image
- It appears that large descriptions brake the page
image

File: db/seeds/prod.rb [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+  conference: conference,
+  title: "Making accessible web apps with Rails and Hotwire",
+  description: "Nowadays, there is a lot of talk about accessibility, but is your web app accessible? In this session, I will share my perspective as a blind developer on how to build accessible web apps with real-world examples, beyond saying it’s important and the need to use ARIA. I will start by demonstrating how a blind person uses a screen reader to navigate the web, so that you can understand the most common challenges faced by blind people every day. This will help clarify several requirements that people currently spend too much time on. Instead, we can focus on the most relevant errors and aspects to bear in mind when developing and designing interfaces. We will cover the most common patterns with code examples, taking advantage of tools provided by Rails, Hotwire, and the browser.",
+  starts_at: Time.zone.local(2024, month, second_day, 13, 45),
+  ends_at: Time.zone.local(2024, month, second_day, 14, 15),
+  location: track_1,
+  speakers: [bruno_prieto_speaker]
+)
+
+Session.create!(
```
[the remark this answers is not in the document]
Comment (andresag4) [reply db/seeds/prod.rb:491]: It's actually on both days!

image
image

File: lib/tasks/prod_seed.rake [hunk 2]
```diff
@@ -0,0 +1,548 @@
...
+    # Create Locations
+    track_1 = conference.locations.find_or_create_by!(name: "Track 1")
+    track_2 = conference.locations.find_or_create_by!(name: "Track 2")
+    lightning_track = conference.locations.find_or_create_by!(name: "Lightning Track")
+
+    # Create Tags
+    # No tags for now
+
+    # Create Speakers
+    matz_speaker = Speaker.find_or_create_by!(name: "Yukihiro \"Matz\" Matsumoto") do |speaker|
```
Comment (Sergio-e) [inline lib/tasks/prod_seed.rake:21]: I forgot that `name` is coming from the profile. I'm getting this error:
```
ActiveRecord::StatementInvalid: SQLite3::SQLException: no such column: speakers.name (ActiveRecord::StatementInvalid)
```

Consider using something like this:
```
    matz_speaker = Profile.find_by(name: "Yukihiro \"Matz\" Matsumoto", profileable: "Speaker")&.speaker
    matz_speaker ||= Speaker.create!(
      name: "Yukihiro \"Matz\" Matsumoto",
      job_title: "Creator of Ruby",
      bio: "Matz is the creator and chief designer of the Ruby programming language.",
      github_url: "https://github.com/matz",
      linkedin_url: "",
      twitter_url: "https://x.com/yukihiro_matz",
      image: Rails.root.join("#{image_path}/m-matsumoto.jpg").open
    )
```

**Pull request merged**